src/components/views/rooms/RoomHeader.js

```diff
@@ -110,7 +110,7 @@ module.exports = React.createClass({
                 var searchStatus;
                 // don't display the search count until the search completes and
                 // gives us a non-null searchCount.
-                if (this.props.searchInfo && this.props.searchInfo.searchCount !== null) {
+                if (this.props.searchInfo && this.props.searchInfo.searchCount != null) {
```

Comment:
We want to distinguish between `0` ('there are no search results') and `undefined` ('we don't yet know if there are any search results').

The comment is out of date, and needs an update.